passes/erreq/erreq.go

```diff
@@ -50,7 +70,56 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			return
 		}
 
-		pass.Reportf(node.Pos(), "do not compare error with \"==\" or \"!=\"")
```

Comment:
Tests didn't pass in my local environment maybe because the format of the report.